table/keys.go

```diff
@@ -30,44 +30,84 @@ type KeyMap struct {
 	ScrollLeft key.Binding
 }
 
-// DefaultKeyMap returns a set of sensible defaults for controlling a focused table.
+// DefaultKeyMap returns a set of sensible defaults for controlling a focused table with help text.
 func DefaultKeyMap() KeyMap {
 	return KeyMap{
 		RowDown: key.NewBinding(
 			key.WithKeys("down", "j"),
+			key.WithHelp("↓/j", "move down"),
 		),
 		RowUp: key.NewBinding(
 			key.WithKeys("up", "k"),
+			key.WithHelp("↑/k", "move up"),
 		),
 		RowSelectToggle: key.NewBinding(
 			key.WithKeys(" ", "enter"),
+			key.WithHelp("<space>/enter", "select row"),
 		),
 		PageDown: key.NewBinding(
 			key.WithKeys("right", "l", "pgdown"),
+			key.WithHelp("→/h/page down", "Previous Page"),
```

Comment:
```suggestion
			key.WithHelp("→/h/page down", "Next Page"),
```

Missed this on first pass, but these are switched.

Reply:
Also as a note, these have inconsistent capitalization with the other text. I don't have a strong opinion on capitalization as long as it's consistent, so maybe just lowercase these?

Reply:
Oh yes, that's just my idiotic habit.
I will make sure all of them are the same as the other ones.